runtime/compiler/codegen/J9RecognizedMethodsEnum.hpp

```diff
@@ -415,6 +415,10 @@
 
    sun_misc_Unsafe_ensureClassInitialized,
 
+#if JAVA_SPEC_VERSION >= 11
```

Comment:
I am not a fan of ifdef'd enum entries this makes corefile analysis and other things much harder than it needs to be - keep the numbering consistent. We do not generally use JDK version macros like this.